src/core/thread/network_data.hpp

```diff
@@ -308,6 +308,16 @@ class NetworkData : public InstanceLocator
     Error GetNextServer(Iterator &aIterator, uint16_t &aRloc16) const;
 
 protected:
+    /**
+     * Defines Service Data match mode.
```

Comment:
```suggestion
     * Define Service Data match mode.
```

Reply:
How about `* This enumeration defines Service Data match mode`.